src/components/tooltip/ToolTip.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Normaltekst, Undertekst } from 'nav-frontend-typografi';
+import { Bold } from '../common-styled-components';
+import ShowThenHide from '../ShowThenHide';
+import { removeWhitespace } from '../../utils/string-utils';
+import * as Cookies from 'js-cookie';
+
+interface Props {
+    children: string;
```

Comment:
Blir det riktig å kalle dette children? Er vel mer "content"? Og bruker vi ikke children stort sett for react sin innebygde children prop?

Reply:
vet ikke helt hva du mener med innebygd children prop, men kan godt ta imot ReactNode istedenfor her